src/core/createLogController.js

```diff
@@ -11,7 +11,7 @@ governing permissions and limitations under the License.
 */
 
 import { queryString, stringToBoolean } from "../utils";
-import logQueryParam from "../constants/logQueryParam";
+import debugQueryParam from "../constants/debugQueryParam";
```

Comment:
Did you forget to add this file?  I don't see it in the tree.

Reply:
I did 😃 . Added it now.